### ISSUE TITLE
Themes

### ISSUE THREAD

**Issue by exerro** [Exerro/Sheets#17]
Application-wide themes which are used by default. Upon sheet creation, its properties will be initialised as `undefined` and use the respective theme value. Any sheet whose properties later change to `undefined` will also use the theme. Themes will be able to be modified during runtime and sheet properties will change dynamically.

Themes will have sections, with each section having a query and a set of properties. That query will be used to match targets, and the valid properties for any sheet that matches the query will be updated unless it explicitly gives a value itself.

**Comment by LoganDark** [comment]
Lua has an `undefined`?

**Comment by exerro** [comment]
Nope. It's something I built into the dynamic property system. Lua-side it's `nil`, but `undefined` is a bit more understandable when it comes to properties, in my opinion.

**Comment by LoganDark** [comment]
Okay
